docs/index.md

```diff
@@ -5,24 +5,49 @@ date: "July 4, 2017"
 output: html_document
 ---
 
-The TCLB code is a parallel Lattice Boltzmann Method solver, able to run on multi-GPU
-cluster architectures.
+TCLB (Templated MPI+CUDA/CPU Lattice Boltzmann Code) is a parallel Lattice Boltzmann Method solver, able to be run on multi-GPU
+cluster architectures.
 
-For a quick start, please refer to the [Getting Started
-guide](/1.-getting-started/installation/).
+For a quick start, please refer to [1. Getting started](/1.-getting-started/installation/).
 
-Detailed descriptions of configuration of the code and compilation, can be
-found in the "Installation" part of this manual. The "Running simulations"
-part details the setup and execution of fluid flow simulations with TCLB.
-The "Post-processing" details the way you can analyze the results. The
-"Model development" part is a reference for developers wanting to add or
-modify models in TCLB.
+## Contents
 
-The "XML Reference" is an automaticaly generated reference of all
-XML elements avaliable for usage in the TCLB configuration files. "Models"
-is also automaticaly generated and is a reference of all models currently
-avaliable, with all apropriate data for each of them.
+1. Getting started
 
-Finnaly "Tutorials" section provides several step-by-step tutorials about
-model development and other things.
+    A quick start guide.
 
+2. Installation
+
+    Detailed descriptions of configuration of the code and compilation.
+
+3. Running simulations
+
+    Details the setup and execution of fluid flow simulations using TCLB.
+
+4. Post-processing
+
+    Details the manner in which the results can be analyzed.
+
+5. Model development
+
+    A reference for developers wanting to add or modify models in TCLB.
+
+6. Contributing
+
+    Details the basics of getting involved with contributing to the TCLB project.
+
+- General Info
+
+    Contains general information related to TCLB, including a list of publications that employ TCLB.
+
+- XML Reference
+
+    An automatically generated reference of all XML elements available for usage in the TCLB configuration files.
+
+- Models
+
+    An automatically generated reference of all models currently available, including all relevant data for each.
+
+- Tutorials
+
+    Provides several step-by-step tutorials on model development.
```

Comment:
These - could be changed to * so that they're bulleted. Based on the preview it seems that these items are numbered, but it would make more sense for them to be bulleted as these items are not numbered in the navigation pane.